### ISSUE TITLE
How to vis training loss via tensorboard?

### ISSUE THREAD

**Issue by ysfalo** [zsdonghao/tensorlayer#46]
In Keras, I just use  
```python
callbacks=[Tensorboard(log_dir)='log']
``` 
in model.fit(), and it can generate training loss in tensorboard automatically. I wonder how can I visualize the training loss via tensorboard in tensorlayer?
